src/Microsoft.QualityGuidelines.Analyzers/Core/MicrosoftQualityGuidelinesAnalyzersResources.resx

```diff
@@ -210,4 +210,13 @@
   <data name="RethrowToPreserveStackDetailsTitle" xml:space="preserve">
     <value>Rethrow to preserve stack details.</value>
   </data>
+  <data name="DoNotRaiseExceptionsInExceptionClausesDescription" xml:space="preserve">
+    <value>When an exception is raised in a finally or fault clause, the new exception hides the active exception. When an exception is raised in a filter clause, the run time silently catches the exception. This makes the original error difficult to detect and debug.</value>
```

Comment:
I've changed the message. Msdn has the rule name as DoNotRaiseExceptionsInExceptionClauses - it's probably better to leave the name as is to so as to avoid any confusion.